remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,13 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, *args):
+        """
+        Storing the Event details using the logger.
+        """
+        event_line = ",".join("{}".format(arg) for arg in args)
+        self.logger.debug(event_line)
```

Comment:
Log doesn't have an `info` method -- remember when you wrote that class @aggFTW ? :)